ogusa/tests/test_aggregates.py

```diff
@@ -73,23 +78,183 @@ def shifted_arr():
     for t in range(T):
         for i in range(s):
             for k in range(j):
-                res_loop[t, i, k] *= (omega_shift[t, i, 0] * imm_shift[t, i, 0] *
-                                      lambdas[k] * b_splus1[t, i, k])
+                res_loop[t, i, k] *= (omega_shift[t, i, 0] * imm_shift[t, i, 0]
+                                      * lambdas[k] * b_splus1[t, i, k])
 
     res_matrix = (b_splus1 * (imm_shift * omega_shift) * lambdas)
 
     assert (np.allclose(res_loop, res_matrix))
 
+    # test SS
     aggI_SS_test = ((1 + g_n[0]) * np.exp(g_y) *
                     (K_p1[0] - res_matrix[0].sum() / (1 + g_n[0])) -
                     (1.0 - delta) * K[0])
     aggI_SS = aggr.get_I(b_splus1[0], K_p1[0], K[0],
-                         (delta, g_y, omega[0], lambdas, imm_rates[0], g_n[0], 'SS'))
+                         (delta, g_y, omega[0], lambdas, imm_rates[0], g_n[0],
+                          'SS'))
     assert (np.allclose(aggI_SS, aggI_SS_test))
 
+    # test TPI
     aggI_TPI_test = ((1 + g_n) * np.exp(g_y) *
                      (K_p1 - res_matrix.sum(1).sum(1) / (1 + g_n)) -
                      (1.0 - delta) * K)
     aggI_TPI = aggr.get_I(b_splus1, K_p1, K,
                           (delta, g_y, omega, lambdas, imm_rates, g_n, 'TPI'))
     assert (np.allclose(aggI_TPI, aggI_TPI_test))
+
+
+def test_get_K():
+    """
+    Simulate data similar to observed
+    """
+    T = 160
+    s, j = 40, 2
+
+    b = -0.1 + (7 * np.random.rand(T * s * j).reshape(T, s, j))
+    omega = 0.5 * np.random.rand(T * s).reshape(T, s, 1)
+    lambdas = 0.4 + (0.2 * np.random.rand(j).reshape(1, 1, j))
```

Comment:
@hdoupe Maybe doesn't matter for the calculations, but `omega.sum()` and `lambdas.sum()` should both be 1 in the model.

Reply:
Ok.  I can normalize `omega` and `lambdas` so that they sum to 1.  How does that sound?

Reply:
If I do normalize omega, which dimensions should sum to 1?